.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.linting.enabled": true,
+    "python.linting.pycodestyleEnabled": true,
```

Comment:
editor-specific settings should not be committed.

Reply:
if we will not commit editor-specific settings such as linter/formatter config, then i guess the other person follows different settings can result into some confusion/difficulty in reading? means i follow 4spaces indent and other person is used to 2 spaces indent. then they have to face issue? By pushing this file, i think everyone has same settings in a particular project?

Reply:
Yes, At the beginning of a project a format is decided and everyone sticks to that, after that settings file is not committed. So in the beginning an agreed upon file is prepared and pushed.

Reply:
That's great approach